core/test/integration/transactions/bitcoin_P2WSH_transaction_tests.cpp

```diff
@@ -28,42 +28,132 @@
  *
  */
 #include "../BaseFixture.h"
-#include "../../fixtures/medium_xpub_fixtures.h"
 #include <wallet/bitcoin/api_impl/BitcoinLikeTransactionApi.h>
 #include <math/bech32/Bech32Factory.h>
 #include "transaction_test_helper.h"
 #include <utils/hex.h>
 #include <api/KeychainEngines.hpp>
+#include "../../fixtures/medium_xpub_fixtures.h"
+#include "../../fixtures/txes_to_wpkh_fixtures.h"
+
 using namespace std;
 
-struct BitcoinMakeP2WSHTransaction : public BitcoinMakeBaseTransaction {
+struct BitcoinMakeTransactionFromNativeSegwitToP2WSH : public BitcoinMakeBaseTransaction {
```

Comment:
A little bit confusing to see `NativeSegwit` to `P2WPKH` (please note the typo "P2WSH") as, for me, `P2WPKH` is actually native segwit (v0). What do you call NativeSegwit ?